deblur/workflow.py

```diff
@@ -417,15 +417,15 @@ def remove_artifacts_seqs(seqs_fp,
 
     if coverage_thresh is None:
         if negate:
-            coverage_thresh = 0.95
+            coverage_thresh = 0.95 * 100
         else:
-            coverage_thresh = 0.3
+            coverage_thresh = 0.6 * 100
```

Comment:
If a sequence doesn't align at 60% of its length to greengenes, do you think it is likely to be 16S given how conserved it is in many regions?